src/CardPage/CardPage.js

```diff
@@ -0,0 +1,25 @@
+import { React, Component } from 'react';
+import './CardPage.scss';
+import { Col, Row, Image } from 'react-bootstrap';
+import { PropTypes } from 'prop-types';
+import { fetchCard } from '../fetch/fetch-requests';
+
+class CardPage extends Component {
```

Comment:
this is looking good, I'd love to refactor it to be a function in stead of a class component